README.md

```diff
@@ -44,7 +44,9 @@ The `linearham`-related command line arguments are described in the following ta
 
 | Command | Description |
 | ---     | ---         |
-| `--cluster-ind` | The `partis` clonal family index (defaults to 0). |
+| `--cluster-ind` | The `partis` clonal family index. |
+| `--partition-ind` | The `partis` clonal family index (defaults to the partition with the highest log probability according to partis). |
```

Comment:
suggest putting the partition index first, and not sure on best wording but they can't both be "clonal family index". Also suggest not shortening "index" to "ind"; I don't think it's obvious what "ind" stands for. Maybe:

partition-index: zero-based index of partition from which to choose a cluster in the partis list of most likely partitions (default: index of most likely partition)
cluster-index: zero-based index of cluster to use in the partition specified by --partition-index (default: 0)